CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Coding guidelines
```

Comment:
I wasn't taking it personally (I think). I was just trying to point out that while I contribute to Rust and I am a C++ expert, that I've seen many people who contribute to Rust who are much closer to the "copy paste code into Mudlet" side of things. In fact, that's the appeal of Rust, the fact that the compiler stops you from getting lots of things wrong. I'd recommend not assuming things about the Rust community when compared to the Mudlet community just because it's a programming language.

Anyways, I'm mostly fine with this, as a start. Hopefully we can expand on it over time.